frontend/src/scenes/surveys/Surveys.tsx

```diff
@@ -86,7 +96,9 @@ export function Surveys(): JSX.Element {
                                 children: 'Configure',
                             }}
                         >
-                            Survey popups are currently disabled for this project.
+                            {usingSurveysSiteApp
+                                ? 'Survey site apps are now deprecated. Enable surveys popup in settings to move to the new system.'
```

Comment:
I know the configure button is right there but I can still see potential confusion so just trying to be as explicit as possible here

```suggestion
                                ? 'Survey site apps are now deprecated. Configure and enable surveys popup in the settings here to move to the new system.'
```

Reply:
Also IMO it might be worth using a "warning" type banner instead of "info" since we _require_ users to do this